tests/test_save_and_restore.py

```diff
@@ -96,3 +98,46 @@ def test_restore_failed_large_bloom_filter(self):
         self.wait_for_logfile(logfile, large_obj_restore_err)
         self.wait_for_logfile(logfile, internal_rdb_err)
         assert not self.server.is_alive()
+
+    def test_rdb_restore_version_compatibility(self):
+        # Create a rdb on bloom-module enabled server
+        bf_server = ValkeyServerHandle(bind_ip=self.get_bind_ip(), port=self.get_bind_port(), port_tracker=self.port_tracker,
+                                        cwd=self.testdir, server_id=self.server_id, server_path=self.server_path)
+        bf_server.set_startup_args({
+            'loadmodule': os.getenv('MODULE_PATH'),
+            'bf.bloom-use-random-seed': self.use_random_seed,
+        })
+        bf_server.start()
+        assert bf_server.is_alive()
+        bf_client = bf_server.get_new_client()
```

Comment:
These aspects of setup (server startup, module load, client connection) are done by default for every new test.

We can remove this.

I understand we need the setup at the current line of code at 125 to start the server without module load.

But the highlighted lines here can be removed, right?
```suggestion
        # Create a rdb on bloom-module enabled server
        bf_server = ValkeyServerHandle(bind_ip=self.get_bind_ip(), port=self.get_bind_port(), port_tracker=self.port_tracker,
                                        cwd=self.testdir, server_id=self.server_id, server_path=self.server_path)
        bf_server.set_startup_args({
            'loadmodule': os.getenv('MODULE_PATH'),
            'bf.bloom-use-random-seed': self.use_random_seed,
        })
        bf_server.start()
        assert bf_server.is_alive()
        bf_client = bf_server.get_new_client()
```